Modix.Bot/Modules/UserInfoModule.cs

```diff
@@ -164,7 +172,10 @@ await ReplyAsync("", embed: new EmbedBuilder()
             timer.Stop();
             embedBuilder.WithFooter(footer => footer.Text = $"Completed after {timer.ElapsedMilliseconds} ms");
 
-            await ReplyAsync(embed: embedBuilder.Build());
+            await _autoRemoveMessageService.RegisterRemovableMessageAsync(
+                userInfo == requesterInfo ? new[] { userInfo } : new[] { userInfo, requesterInfo },
```

Comment:
```suggestion
                userInfo.Id == Context.User.Id ? new[] { userInfo } : new[] { userInfo, requesterInfo },
```
Probably safer to compare IDs, since I'm not sure reference equality will ever return `true` here